qgeotilefetchergooglemaps.cpp

```diff
@@ -49,6 +49,9 @@ QGeoTileFetcherGooglemaps::QGeoTileFetcherGooglemaps(const QVariantMap &paramete
         _language = (langs.length() > 0) ? langs[0] : "en-US";
     }
 
+    if (parameters.contains(QStringLiteral("googlemaps.maps.highdpi")))
+        _scale = (parameters.value(QStringLiteral("googlemaps.maps.highdpi")).toBool())? 2: 1;

```

Comment:
whitespace `.toBool())? 2: 1` ---> `.toBool()) ? 2 : 1`

you need to either set `_scale` to be `1`  in the constructor initializer list or set it here:
```
else
    _scale = 1;
```
otherwise `_scale` would end up being `0` if the parameter is missing from the QML.